dashboard/config/locales/base/en.yml

```diff
@@ -443,6 +443,7 @@ en:
       settings: 'Account settings'
       logout: 'Sign out'
       signin: 'Sign in'
+      signin_cap: 'Sign In'
```

Comment:
Adding this now so we can update in the future when strings come in. See followup ticket: [ACQ-3277](https://codedotorg.atlassian.net/browse/ACQ-3277)

Reply:
I'm definitely missing some context, but won't `text-transform: uppercase;` css rule handle this?

Reply:
I think we decided not to do this with css since we can't guarantee it'll apply correctly to the translated strings. Like if there's a string that comes back like "Word a Word" we don't want the "a" to be capitalized.

Reply:
oh, gotcha, thank you for explaining!